cumulusci/tasks/salesforce/sourcetracking.py

```diff
@@ -0,0 +1,234 @@
+from collections import defaultdict
+import json
+import os
+import re
+import xmltodict
+
+from cumulusci.core.utils import process_bool_arg
+from cumulusci.core.utils import process_list_arg
+from cumulusci.salesforce_api.metadata import ApiRetrieveUnpackaged
+from cumulusci.tasks.salesforce import BaseRetrieveMetadata
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.tasks.metadata.package import PackageXmlGenerator
+from cumulusci.tasks.metadata.package import __location__
+
+
+class ListChanges(BaseSalesforceApiTask):
+
+    task_options = {
+        "include": {"description": "Include changed components matching this string."},
+        "exclude": {"description": "Exclude changed components matching this string."},
+        "exclude_namespace": {"description": "Namespaces that should be excluded."},
+        "snapshot": {
+            "description": "If True, all matching items will be set to be ignored at their current revision number.  This will exclude them from the results unless a new edit is made."
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super(ListChanges, self)._init_options(kwargs)
+        self.options["include"] = process_list_arg(self.options.get("include", []))
+        self.options["exclude"] = process_list_arg(self.options.get("exclude", []))
+        self.options["reset_status"] = process_bool_arg(
+            self.options.get("reset_status", [])
+        )
```

Comment:
Yeah, I think I removed the option because I wanted to limit scope and not implement this.  The idea was to have an easy way to delete the snapshot to reset everything